### PR TITLE
add environment option to createClient

### PR DESCRIPTION
**Pull request by mike-buildable**

### CODE REVIEW

**Review by paulkr — APPROVED** [review]

File: src/apis.ts [hunk 1]
```diff
@@ -2,7 +2,17 @@ import axios from "axios"
...
 import memoize from "lodash.memoize"
 
-export const baseURL = process.env.EVENT_INC_BASE_URL || 'https://development-stream.event.dev';
+export function getUrl(environment?: string) {
+	if (process.env.EVENT_INC_BASE_URL) {
+		return process.env.EVENT_INC_BASE_URL;
+	}
+
+	if (environment === 'production' || process.env.NODE_ENV === 'production') {
+		return 'https://stream.event.dev';
```
Comment (moekatib) [inline src/apis.ts:11]: Should not be hardcoded

File: src/apis.ts [hunk 2]
```diff
@@ -2,7 +2,17 @@ import axios from "axios"
...
+	}
+
+	return 'https://sandbox-stream.event.dev';
```
Comment (moekatib) [inline src/apis.ts:14]: Same ;)

File: src/apis.ts (hunk 1 above)
Comment (moekatib) [reply src/apis.ts:11]: Could be added into a config file and referenced from there

Comment (mike-buildable) [reply src/apis.ts:11]: where will this come from?

Comment (paulkr) [reply src/apis.ts:11]: @moekatib This is how @krish221997 has set up the connections and pipelines packages as well. If we are making a change here, we should do so everywhere

**Pull request merged**